File: tests/eval/java/seen/CleanParameterized.java
import java.sql.*;
public class CleanParameterized {
    void find(Connection c, String name) throws SQLException {
        PreparedStatement st = c.prepareStatement("SELECT * FROM users WHERE name = ?");  // correct: parameterized
        st.setString(1, name);
        st.executeQuery();
    }
}

File: tests/eval/csharp/seen/clean_parameterized.cs
using System.Data.SqlClient;
public class Repo {
    public void Find(SqlConnection c, string q) {
        var cmd = new SqlCommand("SELECT * FROM users WHERE name = @n", c);  // correct: parameterized
        cmd.Parameters.AddWithValue("@n", q);
        cmd.ExecuteReader();
    }
}
